doc/userguide/install.rst

```diff
@@ -165,6 +165,32 @@ Follow these steps from your Suricata directory::
     make -j8 # j is for paralleling, you may de/increase depending on your CPU
     make install # to install your Suricata compiled binary
 
+Auto-Setup
+^^^^^^^^^^
+
+You can also use the available auto-setup features of Suricata:
+
+::
+
+   sudo ./configure && sudo make && sudo make install-conf
```

Comment:
You should only need sudo for the `make install-*` commands.

Reply:
okay, noted. Thank you for the feedback